engine/src/main/java/org/destinationsol/game/screens/MainScreen.java

```diff
@@ -141,6 +143,9 @@ public MainScreen(float resolutionRatio, RightPaneLayout rightPaneLayout, Contex
         controls.add(pauseControl);
         cameraControl = new CameraKeyboardControl(gameOptions, controls);
 
+        consoleControl = new SolUiControl(null, true, Input.Keys.GRAVE);
```

Comment:
Keep in mind `GRAVE` can be a tricky hot key due to different keyboard formats. In Terasology we default to *both* `GRAVE` and `F1` to better catch a wider array of keyboard formats.

Reply:
This is true, I'm constantly annoyed by games having the console on it, as I do not have the key